### PR TITLE
Feature/statdnavi style

### PR DESCRIPTION
**Pull request by SamuelRioTz**

### CODE REVIEW

**Comment by SamuelRioTz** [comment]
@md-weber 
The main container of trufi_core is completely different from Statdnavi, but there are a lot of widgets what we can reuse.
That is why I suggest to create a custom app container for Statdnavi with own blocs, translations and new widgets.
This PR is an example how is working the widgets usage from trufi_core and custom new widgets.
IMG_0140

**Comment by md-weber** [comment]
Mhhh from my perspective, that is not what the Stadtnavi people want, AFAIK they would like to have "Trufi as is in-app" and enhanced with stuff. Therefore our app should only deliver the surrounding and should depend on Trufi Core. Another idea would be to depend on Trufi Core and import the widgets from there. But if we copy and paste it over here, I guess our customer would not be happy with the result because they would need to maintain that in the long run.

I understand that lot of their things are different to our current Core, but instead of creating everything here, I would try to improve the customization of the Trufi Core. For example, you want the two Fabs closer to each other, so we need to provide a possibility to set the position of them in the Trufi Core.

**Review by md-weber — COMMENTED** [review]
Some ideas for future development.

File: lib/screens/HomeScreen.dart [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+}
+
+class HomeScreenState extends State<HomeScreen>
+    with SingleTickerProviderStateMixin {
+  final _formKey = GlobalKey<FormState>();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: _buildAppBar(context),
```
[the remark this answers is not in the document]
Comment (SamuelRioTz) [reply lib/screens/HomeScreen.dart:32]: @md-weber I am completely agree with you, but this PR was just a prove of concept and I just copy-paste original code of trufi_core and disabled the bloc dependencies

**Comment by SamuelRioTz** [comment]
@md-weber The idea is not copy and paste over in this repo> The idea is: the new features keeps on trufi_core, but the exclusive new features keep here.
With "exclusive new features" I mean features what will not be supported by trufi_core but Statdnavi wants.
This scenario can be possible?

**Comment by SamuelRioTz** [comment]
@md-weber @Denis306 
Regarding the UI I am not pretty sure if both app should be equal but just with different colors. Because trufi_core has own base design, Statdnavi too. and both are completely different.
I was thinking to have different UIs sharing common features.

**Comment by md-weber** [comment]
Mhhh, I thought that  If it is interesting for Herrenberg, for example, another city in Germany wants to have the same or close to the same feature list, it would be amazing to enable it in the core. Then it would be maybe event transferrable to a lot of cities in Europe.

Could you share an example that Trufi Core should not support in the long run but is supported by Stadtnavi?

**Comment by SamuelRioTz** [comment]
@md-weber 
I think "should" is not the correct word, I think "could" is the correct word instead.
I am worry about the current existing trufi_core instances will not support the Statdnavi app features, basically because does not exists the concepts.
e.g. the first instance of trufi_core(in my city) the public transport is completely different than Statdnavi, there is no bus stops, no time table, no trains.

**Comment by md-weber** [comment]
This is why we have to work on the Core, as far as I understood, and the idea that I had in mind is to create Feature Toggles that the Host app can activate/deactivate in code (via Config or similar). The Trufi Core enables them or disables them accordingly so the depending app like Trotro, Stadtnavi etc., can pass a set of configurations down to Trufi Core.

As far as I know Trufi Core that was the initial idea and it needs some clean up there, but as I said that is something we should discuss in the daylie tonight.

**Pull request closed without merging**